MapboxGeocoder/MBGeocoder.swift

```diff
@@ -346,6 +356,114 @@ open class Geocoder: NSObject {
         components.queryItems = params
         return components.url!
     }
+    
+    /**
+     Send a keydown event
+    */
+    func initKeydown(options: GeocodeOptions){
```

Comment:
Nit: To avoid confusion, only initializers should begin with “init”. Consider a more descriptive name for this method,